.vscode/tasks.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Auto generated content. I just let VS Code do it automagic and wanted to have the default behavior which is good for most of us.